src/f_finale.c

```diff
@@ -999,7 +1007,7 @@ static const char *credits[] = {
 	"",
 	"\1Sprite Artists",
 	"Odi \"Iceman404\" Atunzu",
-	"Victor \"VAdaPEGA\" Ara\x1Fjo", // Ara�jo -- sorry for our limited font! D:
+	"Victor \"VAdaPEGA\" Ara\x1Fjo", // Araújo -- sorry for our limited font! D:
```

Comment:
What did you change this?

Reply:
For some reason, my editor didn't like the 'ú' character... I tried setting it back and it still thought I changed it. Not sure what happened there.